src/modules/program/dtos/update-program/update-program.dto.ts

```diff
@@ -6,4 +6,10 @@ export class UpdateProgramDto {
 
   @ApiProperty()
   coordinates?: Array<number>
+
+  @ApiProperty()
```

Comment:
Should be mark as optional, base on syntax

Reply:
Fixed